### PR TITLE
Functional tests - Add new test 'FO > Quick view product'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Add new test 'FO > Quick view product'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/06_homePage/04_productQuickView.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        expect(result.coverImage).to.contains(Products.demo_14.coverImage),
+        expect(result.thumbImage).to.contains(Products.demo_14.thumbnailImage),
+      ]);
+    });
+
+    it('should check that \'Add to cart\' button is disabled', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkAddToCartButton', baseContext);
+
+      const isAddToCartButtonDisabled = await homePage.isAddToCartButtonDisabled(page);
+      await expect(isAddToCartButtonDisabled, 'Add to cart button is not disabled').to.be.true;
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/06_homePage/04_productQuickView.js]:
```suggestion
      const isAddToCartButtonEnabled = await homePage.isAddToCartButtonEnabled(page);
      await expect(isAddToCartButtonEnabled, 'Add to cart button is not disabled').to.be.false;
```

File: tests/UI/campaigns/functional/FO/06_homePage/04_productQuickView.js [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+      const isModalVisible = await categoryPage.isQuickViewProductModalVisible(page);
+      await expect(isModalVisible).to.be.true;
+    });
+
+    it('should check that \'Add to cart\' button is disabled', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkAddToCartButton2', baseContext);
+
+      const isAddToCartButtonDisabled = await homePage.isAddToCartButtonDisabled(page);
+      await expect(isAddToCartButtonDisabled, 'Add to cart button is not disabled').to.be.true;
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/06_homePage/04_productQuickView.js]:
```suggestion
      const isAddToCartButtonEnabled = await homePage.isAddToCartButtonEnabled(page);
      await expect(isAddToCartButtonEnabled, 'Add to cart button is not disabled').to.be.false;
```

File: tests/UI/pages/FO/home/index.js [hunk 3]
```diff
@@ -320,7 +404,34 @@ class Home extends FOBasePage {
...
+  }
+
+  /**
+   * Is add to cart button disabled
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  async isAddToCartButtonDisabled(page) {
+    return this.elementVisible(page, `${this.addToCartButton}[disabled]`, 1000);
+  }
```
Comment (Progi1984) [inline tests/UI/pages/FO/home/index.js]:
```suggestion
  /**
   * Is add to cart button enabled
   * @param page {Page} Browser tab
   * @returns {Promise<boolean>}
   */
  async isAddToCartButtonEnabled(page) {
    return !this.elementVisible(page, `${this.addToCartButton}[disabled]`, 1000);
  }
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/pages/FO/home/index.js (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/home/index.js]: Done thx :)

File: tests/UI/campaigns/functional/FO/06_homePage/04_productQuickView.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/FO/06_homePage/04_productQuickView.js]: Done :+1:

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — APPROVED** [review]

**Review by matks — DISMISSED** [review]
Woups apologies I approved it
but build https://github.com/PrestaShop/PrestaShop/actions/runs/3188837948/jobs/5201958237 is failing

**Review by Progi1984 — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Pull request merged**